chain/network/src/store/mod.rs

```diff
@@ -146,3 +141,27 @@ impl Store {
         self.0.iter::<schema::Peers>().collect::<Result<_, _>>().map_err(Error)
     }
 }
+
+impl std::convert::From<near_store::Store> for Store {
```

Comment:
```suggestion
impl From<near_store::Store> for Store {
```

`From` is in prelude.